src/main/kotlin/racingcar/domain/Referee.kt

```diff
@@ -0,0 +1,10 @@
+package racingcar.domain
+
+class Referee {
+    fun getWinner(cars: List<RacingCar>): List<String> {
+        val sortedCars = cars.sortedByDescending { it.movingState }
+        val maxState = sortedCars[0].movingState
```

Comment:
```suggestion
        val maxState = sortedCars.first().movingState
```
first라는 함수를 사용하면 가독성이 좀 더 좋아집니다. 😄